internal/db/cluster/schema.go

```diff
@@ -0,0 +1,61 @@
+package cluster
+
+import "time"
+
+// CreateSchema is the default schema applied when bootstrapping the database.
+const CreateSchema = `
```

Comment:
Don't we want the schema to be assembled through the patches similar to what we're doing in LXD?
Or is that one of the subsequent PRs?

Also, I thought we said we'd prefix the internal table names so that we reduce the likelihood of name conflicts with the consumer?